cli/wrappers/package.json

```diff
@@ -7,10 +7,6 @@
     "bun": "1.1.6",
     "typescript": "^5.3.3"
   },
-  "dependencies": {
-    "@miniflare/kv": "^2.14.1",
-    "@miniflare/storage-file": "^2.14.2"
-  },
   "resolutions": {
```

Comment:
This actually can also be removed since it was an optimization for the miniflare imports. (along with the file it references)